cranelift/filetests/filetests/runtests/cls.clif

```diff
@@ -1,6 +1,7 @@
 test interpret
 test run
 target aarch64
+;; target riscv64 has_b
```

Comment:
It looks like we have lowerings for `cls`. Are these tests disabled for any particular reason?

Reply:
Yes, But My local qemu-riscv64 emulator doesn't implement it, I can't test it. right now I have two implemetion,
One rely on `B` extension , One don't.